source/directives/fjs.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference path="../../bower_components/forms-js/dist/forms-js.d.ts" />
+
+module adaptor.directives {
+
+  export function formsjsDirective($compile, builder) {
+    var formsjsForm:formsjs.Form;
+
+    return {
+      restrict: 'AE',
+      scope: {
+        view: '=',
+        validation: '=',
+        model: '='
+      },
+      controller: ['$scope', function($scope:ng.IScope) {
+        formsjsForm = new formsjs.Form();
+        this.validationService = formsjsForm.validationService;
+        this.registerAttribute = formsjsForm.registerAttribute.bind(formsjsForm);
+      }],
+      link: function(scope, element, attrs) {
+        scope.$watchGroup(['view','validation'], function(values) {
+          var view  = values[0];
+          var validation = values[1];
+          if (view !== undefined && validation !== undefined) {
+
```

Comment:
Yes exactly.

I'd like to support three scenarios:

1. Generate it all <- this is what I've done so far

2. Don't give a view definition but give a validation schema
```html
<form forms-js validation="validation">
<fieldset>
  <legend>Reqnuired fields</legend>
  <fjs-text key="name"></fjs-text>  
  <fjs-text key="email"></fjs-text>
</fieldset>
<fjs-textarea key="comment"></fjs-textarea>
</form>
```

3. Neither, put validation on each field (Not sure about this.. but hey maybe) I guess we would build a validation schema internally.
```html
<form forms-js>
<fieldset>
  <legend>Reqnuired fields</legend>
  <fjs-text min="3" required="true" key="name"></fjs-text>  
  <fjs-text pattern=".+@.+\..+" key="email"></fjs-text>
</fieldset>
<fjs-textarea max="270" key="comment"></fjs-textarea>
</form>
```